WordPress/src/main/java/org/wordpress/android/ui/posts/GutenbergKitActivity.kt

```diff
@@ -585,13 +558,11 @@ class GutenbergKitActivity : BaseAppCompatActivity(), EditorFragmentActivity, Ed
             it.setImageLoader(imageLoader)
         }
 
-        // Ensure that this check happens when post is set
-        setShowGutenbergEditor(savedInstanceState)
```

Comment:
We may need to retain `setShowGutenbergEditor(savedInstanceState)` so the _Use Block Editor_ setting within _My Site_ → _More_ → _Site Settings_ remains functioning. Currently, if you enable GutenbergKit and disable that setting, the block editor is still used for new posts/pages.

Reply:
This _should_ be addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/22151. Note that I haven't been able to review or test that PR just yet. Leaving this note early to make sure I don't forget it.